configurator/backend/handlers/openapi.go

```diff
@@ -1228,3 +1240,19 @@ func (oa *OpenAPI) UpdateUser(ctx *gin.Context, userID string) {
 		ctx.JSON(http.StatusOK, result)
 	}
 }
+
+func (oa *OpenAPI) makeUserPlatformAdmin(ctx *gin.Context, userEmail string, authorizator LocalAuthorizator) error {
+	if ctx.IsAborted() {
```

Comment:
I'm not sure we should check for context abortion here – request execution should be treated like transaction in the sense that it should not be interrupted once started.